port/unix/omrsysinfo.c

```diff
@@ -7751,39 +7779,80 @@ omrsysinfo_get_processes(struct OMRPortLibrary *portLibrary, OMRProcessInfoCallb
 		portLibrary->str_printf(portLibrary, path, sizeof(path), "/proc/%s/cmdline", entry->d_name);
 		file = portLibrary->file_open(portLibrary, path, EsOpenRead, 0);
 		if (file >= 0) {
-			bytes_read = read_fully(portLibrary, file, &command, &buffer_size);
+			bytesRead = read_fully(portLibrary, file, &command, &bufferSize);
 			portLibrary->file_close(portLibrary, file);
 		}
 		/* If cmdline is empty, try reading /proc/[pid]/comm. */
-		if (bytes_read <= 0) {
+		if (bytesRead <= 0) {
 			portLibrary->str_printf(portLibrary, path, sizeof(path), "/proc/%s/comm", entry->d_name);
 			file = portLibrary->file_open(portLibrary, path, EsOpenRead, 0);
 			if (file >= 0) {
-				bytes_read = read_fully(portLibrary, file, &command, &buffer_size);
+				bytesRead = read_fully(portLibrary, file, &command, &bufferSize);
 				portLibrary->file_close(portLibrary, file);
 			}
 		}
 		/* Skip process if no data. */
-		if (bytes_read <= 0) {
+		if (bytesRead <= 0) {
 			continue;
 		}
 		/* Replace null terminators with spaces. */
-		for (i = 0; i < bytes_read; i++) {
+		for (i = 0; i < bytesRead; i++) {
 			if ('\0' == command[i]) {
+#if defined(J9ZOS390) && !defined(OMR_EBCDIC)
+#pragma convlit(suspend)
+#endif /* defined(J9ZOS390) && !defined(OMR_EBCDIC) */
+				/* EBCDIC space to be converted to ASCII later */
 				command[i] = ' ';
+#if defined(J9ZOS390) && !defined(OMR_EBCDIC)
+#pragma convlit(resume)
+#endif /* defined(J9ZOS390) && !defined(OMR_EBCDIC) */
 			}
 		}
-		command[bytes_read] = '\0';
-		/* Call the callback function with PID and command. */
-		callback_result = callback(pid, command, userData);
-		if (0 != callback_result) {
+		command[bytesRead] = '\0';
+#if defined(OMR_EBCDIC) || !defined(J9ZOS390)
+		callbackResult = callback(pid, command, userData);
+#else /* defined(OMR_EBCDIC) || !defined(J9ZOS390) */
+		{
+			intptr_t convertedLen = -1;
+			memset(utf8Command, 0, outputBufferSize);
```

Comment:
`convertedLen` should be declared on line 7822.
I don't think (repeatedly) clearing `utf8Command` is necessary.